components/places/src/storage/bookmarks/public_node.rs

```diff
@@ -478,8 +478,8 @@ mod test {
                 assert_eq!(
                     child.child_guids.unwrap(),
                     &[
-                        SyncGuid("bookmark1___".into()),
-                        SyncGuid("bookmark2___".into())
+                        SyncGuid::from("bookmark1___".to_string()),
```

Comment:
Shouldn't need `to_string()` here. (I don't really care about it in tests, but it would be good to be sure we don't need it)